### DISCUSSION TITLE
Dolby Vision handling regarding Profile and Level

### DISCUSSION THREAD

**Discussion by Dendraspis** [gianni-rosato/svt-av1-psy#63] [Q&A]
Hello,

the only relevant parameter regarding Dolby Vision is `--dolby-vision-rpu`. Unlike other encoders there is no `--dolby-vision-profile` parameter to specify the profile and level, which leads to some questions the docs I have found were not able to answer:
- So how does it work? Is the given RPU metadata read out, calculated and used as is?
- Are the profile and level altered based on other given data? 
- Which profiles and levels are supported?
- How can a specific profile and level be forced?

Can anyone enlighten me, please?

**Comment by mr44er** [comment]
You do finetuning and extract/modify the RPU with https://github.com/quietvoid/dovi_tool

And then with svt-av1-psy:

> * Is the given RPU metadata read out, calculated and used as is?

Yes.
